### ISSUE TITLE
Add index in the `transaction list` in a `block detial page` and turn the order to asc.

### ISSUE THREAD

**Issue by FrederLu** [Magickbase/godwoken_explorer#855]
### By default `ascending` sorting is adopted, and `asc/desc` sorting switching is supported.

<img width="857" alt="image" src="https://user-images.githubusercontent.com/25676576/183870960-22473af8-febf-40cc-b42b-f96be2762ab0.png">

### Similar to the image below.
<img width="852" alt="image" src="https://user-images.githubusercontent.com/25676576/183873815-89887384-7480-415f-b392-dc2b3f2dec43.png">

**Comment by Kirl70** [comment]
In the design draft, there is not enough space to add new sequences. At present, the content of the online page is somewhat inconsistent with the design draft (perhaps the function is not yet perfect).

<img width="747" alt="image" src="https://user-images.githubusercontent.com/106060898/184053745-4818671c-56c1-4514-bfa2-bba4461a9d10.png">

**Comment by Keith-CY** [comment]
> In the design draft, there is not enough space to add new sequences. At present, the content of the online page is somewhat inconsistent with the design draft (perhaps the function is not yet perfect).
> 
> <img alt="image" width="747" src="https://user-images.githubusercontent.com/106060898/184053745-4818671c-56c1-4514-bfa2-bba4461a9d10.png">

Yes, `method` field is not added until this API(https://github.com/Magickbase/godwoken_explorer/issues/762) is ready

We can leave it for now until the `method` is added and have a look at the layout for adding a new field `index`
